cnd/src/network/protocols/lightning_identity.rs

```diff
@@ -1,43 +0,0 @@
-use crate::{network::oneshot_protocol, swap_protocols::SwapId};
-use serde::{Deserialize, Serialize};
-use serde_hex::{SerHex, Strict};
-use serdebug::SerDebug;
-
-/// The message for the Lightning identity sharing protocol.
-#[derive(Clone, Copy, Deserialize, Serialize, SerDebug)]
-pub struct Message {
```

Comment:
This shouldn't be deleted :)

Reply:
I removed the lightning identity on purpose so that it would be included in the main patch - you'd prefer to see it go in with this?  I'm not that fussed actually so instead of waiting for a reply I'll just put it back in :)